### PR TITLE
Fix issues/93 job validation

### PR DESCRIPTION
**Pull request by steakunderscore**
This is the patch written by @AndyObtiva rebased.

### CODE REVIEW

File: app/models/job.rb [hunk 1]
```diff
@@ -1,6 +1,15 @@
 class Job < ActiveRecord::Base
   extend Enumerize
-  STATUSES = [:draft, :published, :archived]
-  enumerize :state, in: STATUSES
+  STATES = %w(draft published archived)
+  enumerize :state, in: STATES
   belongs_to :organization
+  validates :title,
+            presence: true,
+            length: { maximum: MAX_STRING_COLUMN_LENGTH }
```
Comment (steakunderscore) [inline app/models/job.rb]: What are peoples thoughts on having lengths? I don't normally have them, PG and all..

Comment (mathieugagne) [inline app/models/job.rb]: I usually don't do it either but I find it'd be good practice for Rails to validate that it won't blow up at the db level. I'd rather have an error message telling me to reduce my text size than a 500 server error cause PG complained.

Comment (cawel) [inline app/models/job.rb]: I second.

File: app/models/job.rb [hunk 2]
```diff
@@ -1,6 +1,19 @@
 class Job < ActiveRecord::Base
   extend Enumerize
-  STATUSES = [:draft, :published, :archived]
-  enumerize :state, in: STATUSES
+
+  STATES = %w(draft published archived).freeze
```
Comment (steakunderscore) [inline app/models/job.rb]: This should be symbols not strings. So should be using `%i` not `%w`.

**Comment by mathieugagne** [comment]
All good! :+1:

**Pull request merged**